### DISCUSSION TITLE
⭐️⭐️ Star Struck Exchange, I will Star you back⭐️⭐️

### DISCUSSION THREAD

**Discussion by 41pha1** [Schweinepriester/github-profile-achievements#423] [Q&A]
Please give me a star for my repo. I will do the same.

https://github.com/41pha1/MIDI-Converter

Thanks in advance!

**Comment by youtube-midify** [comment]


**Reply by 41pha1** [reply]
Thank you, done as well! <3

**Reply by aarongeo1** [reply]
done https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by 41pha1** [reply]
Done as well!

**Reply by xLevitate** [reply]
@25521 done https://github.com/xLevitate/neptune

**Reply by aarongeo1** [reply]
@25521 done https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by montasim** [reply]
Done @aarongeo1. Please star my repo.
https://github.com/montasim/montasim

**Reply by PARKER-X** [reply]
https://github.com/PARKER-X/ConfidentialRoom/

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad

**Reply by Rantoryu** [reply]
@xLevitate , @aarongeo1 @PARKER-X @bensonarafat I've starred your repos.
Please star mine back :)
https://github.com/Rantoryu/Personal_Website

**Comment by xLevitate** [comment]
done, https://github.com/xLevitate/neptune

**Reply by 41pha1** [reply]
Done as well thanks!

**Reply by montasim** [reply]
Done @xLevitate. Please star my repo.
https://github.com/montasim/montasim

**Comment by drk1rd** [comment]
Done!
https://github.com/drk1rd/drk1rd

**Reply by 41pha1** [reply]
Thank you! Done as well :)

**Reply by Prasham2407** [reply]
Done
https://github.com/Prasham2407/Analog-Clock

**Reply by 41pha1** [reply]
Done thanks!

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad

**Reply by Rantoryu** [reply]
@drk1rd , @Prasham2407 I've starred⭐your repos
Please star mine too. 
https://github.com/Rantoryu/Personal_Website

**Comment by bingyun123** [comment]
Done
[https://github.com/bingyun123/custom_chart_viz](https://github.com/bingyun123/custom_chart_viz)

**Reply by 41pha1** [reply]
Please star my repo before writing done, thank you :)

**Reply by bingyun123** [reply]
sorry, I have re-star it.

**Reply by 41pha1** [reply]
Thanks!

**Reply by montasim** [reply]
Done @41pha1. Please star my repo.
https://github.com/montasim/montasim

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad

**Reply by Rantoryu** [reply]
@bingyun123 I gave You a start. Please star mine too :)
https://github.com/Rantoryu/Personal_Website

**Comment by aarongeo1** [comment]
can you mark this as answer @41pha1

**Comment by Buedenbender** [comment]
> Please give me a star for my repo. I will do the same.
> 
> https://github.com/41pha1/MIDI-Converter
> 
> Thanks in advance!

Done, would you star me back :)

https://github.com/Buedenbender/datscience

**Reply by 41pha1** [reply]
Of course!

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad

**Comment by kensoz** [comment]


**Reply by 41pha1** [reply]
Done!

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad

**Comment by abhinit21** [comment]
Done 👍
Need few more to receive "StarStruck"
https://github.com/abhinit21/abhinit21

**Reply by 41pha1** [reply]
Done 😃

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad

**Comment by abhisheks008** [comment]
Star this repo, https://github.com/abhisheks008/ML-Crate

**Reply by 41pha1** [reply]
Done !

**Reply by abhinit21** [reply]
great repo definitely star ✨everyone

**Comment by 1RB** [comment]
done, https://github.com/1RB/Authworria

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad

**Comment by ArielMAJ** [comment]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by montasim** [reply]
Done @ArielMAJ .

**Reply by ArielMAJ** [reply]
Ty

**Reply by ArielMAJ** [reply]
Done

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad

**Reply by ArielMAJ** [reply]
Already starred you @bensonarafat, waiting for you to star back!

**Reply by bensonarafat** [reply]
done

**Reply by ArielMAJ** [reply]
Tyvm!

**Comment by LauraBigoni** [comment]
Please help me star my repo: https://github.com/LauraBigoni/WeatherApp
I will star you back, just leave the link to your repo too. Thank you guys

**Reply by bensonarafat** [reply]
https://github.com/codepymdev/grad

**Reply by LauraBigoni** [reply]
> https://github.com/codepymdev/grad

thank you very much, i starred your repo back 😄

**Comment by bensonarafat** [comment]
https://github.com/codepymdev/grad

**Reply by bm4578** [reply]
> https://github.com/codepymdev/grad

@bensonarafat done https://github.com/bm4578/dev.git

**Reply by bensonarafat** [reply]
done
